testsuite/framework/src/main/java/org/keycloak/testsuite/util/AdminClientUtil.java

```diff
@@ -109,11 +108,21 @@ public static ResteasyClient createResteasyClient(Boolean followRedirects) {
         return resteasyClientBuilder.build();
     }
 
-    private static SSLContext buildSslContext() {
+    public static SSLContext buildSslContext() {
+        return TestConstants.IS_LISFECYCLE_REMOTE
```

Comment:
IMHO something like using `TestRegistry.INSTANCE.getOrCreateProvider(KeycloakServerProvider.class).createSSLContext()` is slightly better way? For example in case that more implementations would be available, we may not need to update this line. Also no need to create new instance if existing server instance can be re-used etc.

Reply:
Yep, much better, I just did the first thing I thought and I didn't check there was this `getOrCreateProvider` method. Done!